code/modules/admin/verbs/adminhelp.dm

```diff
@@ -79,23 +79,30 @@
 	if(src.handle_spam_prevention(msg,MUTE_ADMINHELP))
 		return
 
+	for(var/datum/adminticket/T in admintickets)
+		if(T.permckey == src.ckey && T.resolved == "No")
```

Comment:
Right but if you say yes the message isn't logged in that ahelp, when perhaps it should?